static/js/dark_theme.js

```diff
@@ -14,6 +16,7 @@ export function enable() {
 
 export function disable() {
     $("body").removeClass("color-scheme-automatic").removeClass("dark-theme");
+    inbox_ui.complete_rerender();
```

Comment:
I would hope that we wouldn't need an explicit rerender when changing theme -- we should be controlling the color scheme via CSS. @madrix01 is there a reason you needed this?

Reply:
I see, it's probably the `correct_stream_color` algorithm. This is probably fine, but needs a comment, or maybe better, we should extract a function `dark_theme.rerender_components_with_javascript_color_logic()` that is just a wrapper for `inbox_ui.complete_rerender`